blivetgui/dialogs/constants.py

```diff
@@ -0,0 +1,5 @@
+
+class DialogResponseType(object):
+    BACK = 1
+    REPORT = 2
+    QUIT = 3
```

Comment:
Because Gtk needs `int`, not `Enum` members. (And using `BACK.value` everywhere seemed too complicated to me.)